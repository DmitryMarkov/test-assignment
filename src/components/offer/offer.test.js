import React from 'react'
import { render } from '@testing-library/react'
import { BrowserRouter as Router } from 'react-router-dom'
import Offer from './'

const withRouter = ({ children }) => <Router>{children}</Router>

test('Offer should render properly', async () => {
  const { asFragment, getByText } = render(<Offer />, { wrapper: withRouter })
  const headingTitle = getByText(/Details page for id:/)

  expect(headingTitle).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()
})
