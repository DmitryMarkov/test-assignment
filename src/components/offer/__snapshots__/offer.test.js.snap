// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Offer should render properly 1`] = `
<DocumentFragment>
  <h1
    class="css-15icvz4"
    data-testid="heading-title"
  >
    Details page for id: 
  </h1>
  <div
    class="css-1nvlnqn"
  >
    Details to be done
  </div>
  <button
    class="css-1ri1vrn"
  >
    Go Back
  </button>
</DocumentFragment>
`;
